app/app/src/main/java/org/omecproject/up4/impl/Up4DeviceManager.java

```diff
@@ -432,16 +432,24 @@ private void ensureInterfacesInstalled() {
     public Collection<UpfInterface> configInterfaces() {
         Collection<UpfInterface> interfaces = new ArrayList<>();
         if (config.n3Address().isPresent()) {
-            interfaces.add(UpfInterface.createN3From(config.n3Address().get(), SLICE_MOBILE));
+            if (config.sliceId().isPresent()) {
+                interfaces.add(UpfInterface.createN3From(config.n3Address().get(), config.sliceId().get()));
+            } else {
+                log.error("Missing slice ID in the netcfg to configure N6 interface for {}", config.n3Address().get());
+            }
         }
         for (Ip4Prefix uePool : config.uePools()) {
-            interfaces.add(UpfInterface.createUePoolFrom(uePool, SLICE_MOBILE));
+            if (config.sliceId().isPresent()) {
+                interfaces.add(UpfInterface.createUePoolFrom(uePool, config.sliceId().get()));
+            } else {
+                log.error("Missing slice ID in the netcfg to configure N3 interface for {}", uePool);
+            }
         }
         Ip4Address dbufDrainAddr = config.dbufDrainAddr();
         if (dbufDrainAddr != null) {
             // TODO: change slice ID when adding multi-slice support,
             //  see: https://jira.opennetworking.org/browse/SDFAB-986
-            interfaces.add(UpfInterface.createDbufReceiverFrom(dbufDrainAddr, SLICE_MOBILE));
+            interfaces.add(UpfInterface.createDbufReceiverFrom(dbufDrainAddr, DEFAULT_SLICE_ID));
```

Comment:
I'm not sure about this, should we allow for configuring this via netcfg?

Reply:
What could go wrong if we leave the default?

Reply:
I believe there is a chance we could send traffic the the system Q

Reply:
It's ok using the default for now, we won't have any real-time traffic in P4-UPF for Aether 2.0.